specifyweb/frontend/js_src/lib/components/InitialContext/index.ts

```diff
@@ -100,6 +100,8 @@ export const initialContext = Promise.all([
   import('./userInformation'),
   // Fetch user permissions (NOT CACHED)
   import('../Permissions'),
+  // Fetch the discipline's uniquenessRules (NOT CACHED)
+  import('../DataModel/uniquenessRules'),
```

Comment:
(optional) should we cache this and bust the cache on rule changes (for current browser only - not ideal but might be worth the trade-off)?

or is that too dangerous because rules might have been edited in the meanwhile by someone else?

should we cache it at least for 24hrs then or something short like that?

(don't think people would be changing uniqueness rules all the time, so no need to delay every front-end load, and send more requests to the back-end if we don't have to)